task-2/Program.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+
+namespace task_2
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine(@"Hi there! I'm the Square calculator
+Enter the side's length of your square in integer:");
+
+            int squareLength = int.Parse(Console.ReadLine());
+            PerimeterCalculator(squareLength);
+            AreaCalculator(squareLength);
+            Console.WriteLine(
+                "The perimeter is " +
+                PerimeterCalculator(squareLength) +
+                " and the area is " +
+                AreaCalculator(squareLength)
+                );
+            Console.WriteLine(@"
+////Type to exit");
+            Console.ReadKey();
+
+        }
+
+        static int PerimeterCalculator(int arg)
```

Comment:
Имя метода лучше придумывать такое, чтобы оно начиналась с глагола (метод - это действие!). Имя параметра метода тоже лучше придумать значимое (а не абстрактное - аргумент). Например
```suggestion
        static double CalculatePerimeter(double side)
```
или
```suggestion
        static double GetPerimeter(double side)
```